l1-contracts/contracts/state-transition/verifiers/TestnetVerifier.sol

```diff
@@ -2,30 +2,43 @@
 
 pragma solidity 0.8.28;
 
-import {DualVerifier} from "./DualVerifier.sol";
+import {L1VerifierFflonk} from "./L1VerifierFflonk.sol";
 import {IVerifierV2} from "../chain-interfaces/IVerifierV2.sol";
 import {IVerifier} from "../chain-interfaces/IVerifier.sol";
 
 /// @author Matter Labs
 /// @custom:security-contact security@matterlabs.dev
 /// @notice Modified version of the main verifier contract for the testnet environment
 /// @dev This contract is used to skip the zkp verification for the testnet environment.
-/// If the proof is not empty, it will verify it using the main verifier contract,
+/// If the proof is not empty, it will verify it using the FFLONK verifier,
 /// otherwise, it will skip the verification.
-contract TestnetVerifier is DualVerifier {
-    constructor(IVerifierV2 _fflonkVerifier, IVerifier _plonkVerifier) DualVerifier(_fflonkVerifier, _plonkVerifier) {
+contract TestnetVerifier is IVerifierV2, IVerifier {
+    L1VerifierFflonk private immutable fflonkVerifier;
```

Comment:
Maybe using inheritance would be a bit easier here? One wouldnt have to deploy fflonk verifier separately / number of deployed contracts would be same on testnet/mainnet